addons/website/controllers/main.py

```diff
@@ -320,6 +321,13 @@ def website_configurator(self, step=1, **kwargs):
             action_url += '&step=' + str(step)
         return request.redirect(action_url)
 
+    @http.route('/website/background_installation', type='json', auth='user', website=True)
+    def background_installation(self):
+        # wait a little bit to avoid blocking assets loading in website editor
+        time.sleep(10)
```

Comment:
This should be fixed on js side instead (i.e. make request only when  page is fully loaded)